core/tests/test_utils.py

```diff
@@ -682,10 +802,13 @@ def urlfetch_mock(
         requests to fetch the Gravatar profile picture for new users while the
         backend tests are being run.
 
-        args:
-          - content: Response content or body.
-          - status_code: Response status code.
-          - headers: Response headers.
+        Args:
+            content: str. Response content or body.
+            status_code: int. Response status code.
+            headers: dict. The headers in subsequent calls to the
+            urlfetch mock. The keys of this dict are strings that represent
```

Comment:
Indent continuation lines by 4. Follow the docstring suggestions given for set_return_values() above.

Reply:
Done!